health/micro-ui-react19/web/packages/modules/health-dss/src/configs/UICustomizations.js

```
@@ -0,0 +1,540 @@
+            return data?.MdmsRes?.["HCM-PROJECT-TYPES"]?.projectTypes;
+          },
+        },
+      };
+    },
+    additionalCustomizations: (row, key, column, value, t, searchResult) => {
+      switch (key) {
+        case "ACTIONS":
+          let options = [
+            { code: "1", name: "VIEW_DASHBOARD" },
+            { code: "2", name: "VIEW_CUSTOM_REPORT" },
+          ];
```

Biome (2.5.0)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)